extractor/src/test/suite/scala/com/digitalasset/extractor/TransactionMultiTableSpec.scala

```diff
@@ -49,14 +52,13 @@ class TransactionMultiTableSpec
         case TransactionResult(
               transaction_id,
               seq,
-              workflow_id,
+              _, // Daml Script leaves the workflow identifier empty
```

Comment:
Daml Script doesn't use the workflow identifier like scenarios apparently, so I ignored it in all tests.

Reply:
at some point I’ll understand what the workflow id is but that time is not today

Reply:
I think that the idea behind it is to correlate transactions within a single logical workflow, e.g. if you have some form of trade that needs to be separated in multiple legs due to some business requirement.